PWGHF/Utils/utilsEvSelHf.h

```diff
@@ -323,14 +323,11 @@ struct HfEventSelectionMc {
           centrality = collCent;
           multiplicity = collMult;
         }
-      } else {
-        LOGP(fatal, "Unsupported centrality estimator!");
       }
-    }
-
-    /// centrality selection
-    if (centrality < centralityMin || centrality > centralityMax) {
-      SETBIT(rejectionMask, EventRejection::Centrality);
+      /// centrality selection
+      if (centrality < centralityMin || centrality > centralityMax) {
+        SETBIT(rejectionMask, EventRejection::Centrality);
+      }
```

Comment:
maybe apply it only `if constexpr (centEstimator != o2::hf_centrality::CentralityEstimator::None)`